source/main/gameplay/BeamEngine.cpp

```diff
@@ -248,7 +291,7 @@ void BeamEngine::update(float dt, int doUpdate)
 	if (automode < MANUAL)
 	{
 		// auto-shift
-		if (shifting)
+		if (shifting && !is_Electric) //No shifting in electric cars

```

Comment:
I don't like this at all, why shouldn't an electric vehicle have a transmission? if you wanted to have no shifting then I would make a transmission with 1 gear and set the shift RPM to the max RPM or higher.